src/main/java/seedu/trippie/Parser.java

```diff
@@ -39,6 +39,8 @@ public static Command parse(String userInput) {
                 return new HelpCommand();
             } else if (userInput.startsWith("list /p")) {
                 return new ListPlacesCommand(userInput);
+            } else if (userInput.startsWith("search")) {
```

Comment:
It supposed to be `userInput.startsWith("search ")`. If not, input like `searchex` will be detected as valid command.